app/client/cli/debug.go

```diff
@@ -5,6 +5,10 @@ import (
 	"os"
 
 	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"github.com/pokt-network/pocket/libp2p"
```

Comment:
Not actionable: 💯 I like this. I guess we should start doing this across the board and separate external/internal imports. Maybe we should create an issue to tackle it in one scoop also to create team awareness since it's gonna be discussed at least once in standup at some point. 

Probably too boring to be outsourced to the community 🤔 and however low priority, something to do to kill boredom with more... boredom 😆. Wdyt?

Reply:
I have summoned the community in #185